README.md

```diff
@@ -17,27 +17,62 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
 
 This repository acts as the source code location for [the Unicorn Utterances blog](https://unicorn-utterances.com)
 
-## Important Files
+## Sponsors
+
+[![The Polyglot Developer](./static/sponsors/the-polyglot-developer.svg)](https://www.thepolyglotdeveloper.com/)
+
+## Statement of Ethics
+
+We never want to end up in a place where our educational content or the community or
```

Comment:
One of the concerns raised by @evelynhathaway:
> I would rephrase the "never [negative thing]" to "always [positive thing saying outright why it's good and hinting at why it's bad]"

Reply:
I would argue that in this particular instance
I think with ethics hard lines do need to be drawn occationally
And I think laying it out as the first item showcases that

Reply:
@crutchcorn, the above is my only note after reviewing the entire PR. If no one else agreed, it's resolved in my book.

I'm sorry I couldn't make the review window. I am glad we're doing this. 😄